addons/mayastor/disable

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+echo "Disabling Mayastor"
+
+# Delete the resources
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/operator-rbac.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor-control-plane/master/deploy/mayastorpoolcrd.yaml --force || true
+$KUBECTL delete -f https://raw.githubusercontent.com/openebs/mayastor/master/deploy/nats-deployment.yaml --force || true
```

Comment:
@AlexsJones We are planning to remove the nats in the next release, So if we add this to snap store now, will removing it in the next release be an issue?